code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -128,7 +128,7 @@
 	desc = "A card used to provide ID and determine access across the station."
 	icon_state = "id"
 	item_state = "card-id"
-	var/access = list()
+	var/list/access = list()

```

Comment:
Yep i'm sure. it's treated like a list everywhere.
ill fix any problems that result from this